src/commons/big_query/streaming/data_streamer.py

```diff
@@ -0,0 +1,56 @@
+import datetime
+import json
+import logging
+import uuid
+
+from apiclient.errors import Error
+import httplib2
+import googleapiclient.discovery
+from oauth2client.client import GoogleCredentials
+
+from src.commons.big_query.big_query_table import BigQueryTable
+from src.commons.decorators.retry import retry
+
+
+class DataStreamer(object):
+
+    @staticmethod
+    def _create_http():
+        return httplib2.Http(timeout=60)
+
+    def __init__(self, project_id, dataset_id, table_id):
+        self.big_query_table = BigQueryTable(project_id, dataset_id, table_id)
+        self.http = self._create_http()
+        self.service = googleapiclient.discovery.build(
+            'bigquery',
+            'v2',
+            credentials=GoogleCredentials.get_application_default(),
+            http=self.http
+        )
+
+    def stream_stats(self, rows, insert_id=None):
+        if insert_id is None: insert_id = uuid.uuid4()
```

Comment:
IntelliJ yells to make new line after colon. Anyway, I found that it's equivalent to this:
`insert_id = insert_id or uuid.uuid4()`

Reply:
Done